src/io/flutter/actions/FlutterAppAction.java

```diff
@@ -19,8 +19,8 @@
 abstract public class FlutterAppAction extends DumbAwareAction {
   private static final Logger LOG = Logger.getInstance(FlutterAppAction.class);
 
-  private final @NotNull FlutterApp myApp;
-  private final Computable<Boolean> myIsApplicable;
+  @NotNull private final FlutterApp myApp;
```

Comment:
For the method declaration below it did, but these field declarations it did not.